packages/shared/src/components/Feed.spec.tsx

```diff
@@ -1058,3 +1073,126 @@ it('should report irrelevant tags', async () => {
     ).not.toBeInTheDocument(),
   );
 });
+
+describe('acquisition form card', () => {
+  const replaceRouter = jest.fn();
+  const url = new URL(`http://localhost:5002?${acquisitionKey}=true`);
+
+  let mockUserAcquisitionVersion = UserAcquisition.V1;
+
+  beforeEach(() => {
+    /* eslint-disable @typescript-eslint/no-var-requires,global-require */
+    const useRouter = jest.spyOn(require('next/router'), 'useRouter');
```

Comment:
U can use this way to mock useRouter to avoid the above eslint issues - https://github.com/dailydotdev/apps/blob/dfa97065f51a017db5babf2f8bda03570c6f5dda/packages/shared/src/components/SidebarRankProgress.spec.tsx#L34